canister/candid.did

```diff
@@ -88,6 +88,7 @@ type set_config_request = record {
   fees : opt fees;
   api_access : opt flag;
   disable_api_if_not_fully_synced : opt flag;
+  watchdog_canister : opt principal;
```

Comment:
Shouldn't this be `opt opt principal`? Otherwise, it's not matching the Rust type.

Reply:
Right, thank you! Done.